.github/scripts/config-file-server.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Generates config files in PandABlocks-server and server repositories and populates them with information.
+
+PLATFORM=$1
+
+# Determine the toolchain to use
+if [ "$PLATFORM" == "zynq" ]; then
+    TOOLCHAIN=gcc-arm-10.2-2020.11-x86_64-arm-none-linux-gnueabihf
+elif [ "$PLATFORM" == "zynqmp" ]; then
+    TOOLCHAIN=gcc-arm-10.2-2020.11-x86_64-aarch64-none-linux-gnu
+fi
```

Comment:
What if `$PLATFORM` doesn't match?

Reply:
Added else condition to report mismatch and halt workflow